package.json

```diff
@@ -5,31 +5,13 @@
   "main": "index.js",
   "scripts": {
     "test:open": "npm run lint && cypress open",
-    "pretest": "rm -rf cypress/reports/*",
-    "test": "cypress run",
-    "posttest": "npm run combine:reports && npm run generate:reports",
     "lint": "eslint ./cypress/e2e/**/*.ts",
     "lint:fix": "npm run lint -- --fix",
-    "combine:reports": "mochawesome-merge cypress/reports/mocha/*.json > cypress/reports/report.json",
-    "generate:reports": "marge cypress/reports/report.json -f report -o cypress/reports"
+    "test": "cypress run"
+
```

Comment:
Something happened with the package.json?

Reply:
Yes, I have problems run the test with mocha reports but Its resolved now